class Exception0{

   public static void main(String[] args){

    try{
         int a = 100/0;
    }
    catch(Exception e){
      System.out.println("it can't divide because it will goes infinity");
    }
  }
}